curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4a9.md

```diff
@@ -0,0 +1,32 @@
+---
+id: 66c8ba975ee7230e29f6c4a9
+title: Step 2
+challengeType: 1
+dashedName: step-2
+---
+
+# --description--
+
+The next thing to do is to create a variable for the minimum amount of `annualIncome` and `creditScore` required to be qualified for any of duplex, car, and condo loans.
+
+Use `const` to create a `minIncomeForDuplex` variable set to `60000`, and `minCreditScoreForDuplex` variables set to `700`
```

Comment:
for here, I don't think we should care that much if they use `const`, `let`, or `var` really.
I think the only time we should care, is when we need them to reassign values. Then we should specify `let`.
Otherwise, we should only care that they created the variables. 

The lecture videos will be able to go into depth on when to use `let` and `const` and the differences between them.